fastai/vision/data.py

```diff
@@ -112,7 +112,11 @@ def from_path_func(cls, path, fnames, label_func, valid_pct=0.2, seed=None, item
         return cls.from_dblock(dblock, fnames, path=path, **kwargs)
 
     @classmethod
-    def from_name_func(cls, path, fnames, label_func, **kwargs):
+    def from_name_func(cls,
+                       path:(str, Path), # Set the default path to a directory that a `learner` can use to save files like models.
+                       fnames, # A list of `pathlib.Path` to individual image files.
+                       label_func:callable, # A function that receives a string (the file name) and outputs a label.
+                       **kwargs) -> DataLoaders:
```

Comment:
```suggestion
    def from_name_func(cls,
                       path:(str, Path), # Set the default path to a directory that a `Learner` can use to save files like models
                       fnames:list, # A list of `os.Pathlike`'s to individual image files
                       label_func:callable, # A function that receives a string (the file name) and outputs a label
                       **kwargs
                   ) -> DataLoaders:
```
Formatting nit + typing